float64[] stiffness
---
int8 err
